spacy_llm/tasks/ner.py

```diff
@@ -114,3 +115,9 @@ def assign_spans(
     ) -> None:
         """Assign spans to the document."""
         doc.set_ents(filter_spans(spans))
+
+    def scorer(
+        self,
+        examples: Iterable[Example],
+    ) -> Dict[str, Any]:
+        return get_ner_prf(examples, attr="ents")
```

Comment:
```suggestion
        return get_ner_prf(examples)
```

Reply:
@bdura Let's accept this (if you don't see a reason not to), then I'll merge.